### ISSUE TITLE
Tooltip content disappears when the pointer is moved away from the trigger

### ISSUE THREAD

**Issue by lloydstubber** [cultureamp/kaizen-design-system#1923]
TLDR; Mentioned in their recommendation

Type: WCAG 2
Principle: Perceivable
Guideline:1.4 Distinguishable
Success Criteria: 1.4.13 Content on Hover or Focus (AA)
Level: AA
Platform: Web
Severity: High

### What's the issue?
On the shared agenda tooltip, the pop up disappears when the pointer moves away from the trigger.

<img width="651" alt="Screen Shot 2021-09-02 at 4 25 01 pm" src="https://user-images.githubusercontent.com/18339250/131793027-d927e055-e73d-4c1e-95e4-b2d597608089.png">

### Why it matters
People with fine motor control disabilities, for example essential tremor (ET) or Parkinson disease, are at higher risk of accidentally dismissing the content before they intended.

### Our recommendation
Do not dismiss the tooltip pop up while the pointer is located over the trigger element or the triggered content. A user should be able to move their mouse from the trigger and into the triggered content without it disappearing.

**Comment by lloydstubber** [comment]
Closing this as the popover is very barebones and the interaction is up to the implementation. So this falls outside of what we can currently do for the audit without creating a better component.
